source/shared/net/guides/multisig-transfer.rst

```diff
@@ -43,23 +41,23 @@ Make a transfer proposal
 
 #.  Select **Send GTU with a schedule**.
 
-#.  Select the relevant proposer. You’ll see a list of all the identities that you’re the custodian of. When you select an identity, the **Transactions Details** in the left pane are updated with the identity information and the current transaction expiry time. You'll be able to change the expiry time later in this process. Use the scrollbar if you can't see all the information. Select **Continue**.
-
-#. Select the relevant sender account from the list of available accounts, and then select **Continue**.
+#. All multi-signature accounts are displayed. Select the relevant sender account from the list of available accounts, and then select **Continue**. You'll be able to change the expiry time later in this process. Use the scrollbar if you can't see all the information. Select **Continue**.
```

Comment:
```suggestion
#. All multi-signature accounts are displayed. Select the relevant sender account from the list of available accounts, and then select **Continue**. If you only have one multi-signature account available, this will be chosen automatically. You'll be able to change the expiry time later in this process. Use the scrollbar if you can't see all the information. Select **Continue**.
```